Лабораторная работа №1/Замена пропущенного элемента средним арифметическим.py

```diff
@@ -0,0 +1,7 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+index_of_None = 4
+sum_without_None = sum(numbers[:index_of_None] + numbers[index_of_None+1:])
+count_of_None = len(numbers)
+average_value = sum_without_None/count_of_None
```

Comment:
Не забывайте разделять пробелами переменные и математические операторы. 
Также в последующем вы можете объединять строки, выполняя больше математических операций за раз. Учтите это в последующих лабораторных работах
```suggestion
average_value = sum(numbers[:index_of_None] + numbers[index_of_None+1:]) / len(numbers)
```